src/controllers/product.controller.js

```diff
@@ -0,0 +1,23 @@
+import { productsCollection, usersCollection, sessionsCollection} from "../database/db.js";
+
+export async function getWines(req, res){
+
+    try{
+        
+        const token = req.token;
+
+        const wines = await productsCollection.find().toArray();
+
+        const sessions = await sessionsCollection.findOne({token});
```

Comment:
Eu vou comentar aqui mas só pra ajustar depois do merge mesmo. Acho que seria bom chamar essa session e ver se ela está ativa no middleware do token. Digo no caso de um usuário com token expirado estar tentando ver os produtos, aí deve dar um erro por conta disso e acabar caindo no "server not running".